influxdb3_server/src/http/v1.rs

```diff
@@ -58,16 +59,23 @@ where
             query,
         } = params;
 
+        let format = QueryFormat::from_request(&req, pretty)?;
+        info!(?format, "handle v1 format API");
```

Comment:
I think the message should be `"handle v1 query API"`

Reply:
There is already a log statement "handle v1 query API" at line 52:
 `info!(?params, "handle v1 query API");` (see [line 52](https://github.com/influxdata/influxdb/pull/25030/files/f6050813833396e362aa8b2588f913426a6d60d1#diff-16fe6f4a80676302f95880f0a418000db6011e1de5dfa0180725abca77797f6cR52)).

The purpose of this log was to include the query format as well.